src/screens/index.js

```diff
@@ -24,7 +24,7 @@ export * from './LunchScreen';
 export * from './MapViewScreen';
 export * from './MultiButtonScreen';
 export * from './NestedInfoScreen';
-export * from './NoticeboardIndexScreen';
+export * from './Noticeboard';
```

Comment:
folders are exported at the top of the file. i will move it

Reply:
601e6f4

Reply:
Yes, that's right. Sorry.